.circleci/config.yml

```diff
@@ -676,7 +708,7 @@ workflows:
             - eunit
             - static_analysis
             - osx_package
-            - hodl
+            - hodl_blue
```

Comment:
As blue is deployed before green, this means that macOS binary uploaded before green deployed. Ok. (Just confirming.)